.github/workflows/deploy-to-preproduction.yml

```
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     environment:
       name: preproduction
-      url: ${{ steps.deployment.outputs.preview-url }}
+      url: ${{ secrets.PREPRODUCTION_URL }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)